samples/stock/demo/rangeselector-datagrouping/demo.details

```diff
@@ -0,0 +1,10 @@
+---
+name: Rangeselector Datagrouping
```

Comment:
Other demos have natural language title, so I think this should be:

"Range selector with data grouping"